docs/topics/development/acl.md

```diff
@@ -0,0 +1,34 @@
+(acl)=
+
+# Access Control Lists
+
+## How permissions work
+
+On top of that we use the `access.models.GroupUser` and `Group` to define
+what access groups a user is a part of, and each group has `rules` defining
+which permissions they grant their members, separated by `,`.
+
+Permissions that you can use as filters can be either explicit or general.
+
+For example `Admin:EditAddons` means only someone with that permission will
+validate.
+
+If you simply require that a user has `some` permission in the `Admin` group
```

Comment:
in the RST this sentence is rendered as:
"If you simply require that a user has _some_ permission in the _Admin_ group you can use `Admin:%`. "
and in markdown:
"If you simply require that a user has `some `permission in the `Admin `group you can use `Admin:%`. "
i.e. the convertor is treating single backtick the same as double backtick, looks like.

Reply:
I guess this converter isn't all that great after all. Maybe I'll try feeding the originals into chatGPT and see if it does better 🙈